relational/process_schemaless.py

```diff
@@ -104,9 +108,10 @@ def store_seen_id(row, header, seen_set):
 ])
 
 
-_FIELDS_TO_CHECK = defaultdict(set)
-_FIELDS_TO_CHECK['ppts'].add('date_opened')
-_FIELDS_TO_CHECK['ppts'].add('date_closed')
+_FIELD_PREDICATE = {}
+_FIELD_PREDICATE[PPTS.NAME] = lambda field: \
+        field == 'date_opened' or field == 'date_closed'
+_FIELD_PREDICATE[PTS.NAME] = lambda field: re.search('date', field)
```

Comment:
Nit: Don't feel strongly but it feels a little safer to just have an explicit of the exact fields we are looking for?

Reply:
sounds good - i'll address this in a follow-up commit